go.mod

```diff
@@ -5,25 +5,26 @@ go 1.23.1
 require (
 	github.com/pingcap/errors v0.11.5-0.20221009092201-b66cddb77c32
 	github.com/sirupsen/logrus v1.9.3
-	github.com/stretchr/testify v1.9.0 // indirect
+	github.com/stretchr/testify v1.9.0
 )
 
 require (
 	github.com/alecthomas/chroma v0.10.0
 	github.com/jstemmer/go-junit-report/v2 v2.1.0
 	github.com/olekukonko/tablewriter v0.0.5
-	golang.org/x/term v0.22.0
-	vitess.io/vitess v0.10.3-0.20240709144253-eb29999a3f47
+	github.com/spf13/cobra v1.8.1
+	golang.org/x/term v0.24.0
+	vitess.io/vitess v0.10.3-0.20241001054030-978c59d80f4d
```

Comment:
@systay Yay! This update I think also should resolve the error in https://github.com/vitessio/vitess/actions/runs/11146383094/job/30979150349?pr=16493 which is entirely unrelated to the other changes here, but a good thing to have fixed and would help making that other PR green :smile:.